client/src/app/core/actions/motion-action.ts

```diff
@@ -25,30 +34,29 @@ export namespace MotionAction {
         number?: string;
         sort_parent_id?: Id;
 
-        // Special logic: If lead_motion_id is given, it is an amendment.
-        // Either text or amendment_paragraph_* must be set. Text must be set (required!),
-        // if either lead_motion_id is not given or statute_paragraph_id is given. But I'm not sure here.
-        text: UnsafeHtml;
-        amendment_paragraph?: UnsafeHtml;
-        // lead_motion_id?: Id;
         statute_paragraph_id?: Id;
         reason?: UnsafeHtml; // is required, if special settings are set
-        origin_id: Id;
+        origin_id?: Id;
 
         // Special field, not in model
-        workflow_id: Id;
+        workflow_id?: Id;
 
         // Special field: The content is different as retrieving a mtion
-        submitter_ids: Id[]; // User ids!!! Not the motion_submitter through-model ids!!!
+        submitter_ids?: Id[]; // User ids!!! Not the motion_submitter through-model ids!!!
+    }
+
+    export interface CreatePayload extends PartialMotionPayload {
+        // Special logic: If lead_motion_id is given, it is an amendment.
+        // Either text or amendment_paragraph_* must be set. Text must be set (required!),
+        // if either lead_motion_id is not given or statute_paragraph_id is given. But I'm not sure here.
+        text: UnsafeHtml;
+    }
 
-        // Non-model fields for customizing the agenda item creation, optional
-        agenda_create: boolean;
-        agenda_type: number;
-        agenda_parent_id: number;
-        agenda_comment: string;
-        agenda_duration: number;
-        agenda_weight: number;
+    export interface CreateAmendmentPayload extends PartialMotionPayload {
+        lead_motion_id: Id;
+        [paragraph_number: number]: UnsafeHtml;
```

Comment:
This does not look right. I guess you are meaning:
```
lead_motion_id: Id;
amendment_paragraphs: {
    [paragraph_number]: UnsafeHtml:
}
```

Also: It is either `text` or `amendment_paragraphs`, see [the comment here](https://github.com/OpenSlides-action-crew/openslides-backend-coordination/issues/24#issuecomment-717050417)